NuKeeper.Github/Mappings/AutoMapperConfiguration.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using AutoMapper;
+using NuKeeper.Abstract;
+using NuKeeper.Github.Engine;
+using Octokit;
+
+namespace NuKeeper.Github.Mappings
+{
+    public static class AutoMapperConfiguration
+    {
+        public static IMapper GithubMappingConfiguration { get; }
+
+        [SuppressMessage("ReSharper", "CA1810")]
+        static AutoMapperConfiguration()
+        {
+            var mapper = new Lazy<MapperConfiguration>(()=>
+                new MapperConfiguration(cfg =>
```

Comment:
For this to be actually lazy then I think you need to put the `CreateMapper` call inside of the `Lazy` and the `.Value` part should go in the property getter.

This looks like you are eagerly creating it the first time and setting the static property that value.

Reply:
Why do we have automapper introduced in the first place?